content/governance.md

```diff
@@ -4,8 +4,7 @@ title = "Governance"
 
 ## Organization for Ethical Source
 
-* Bylaws ([English](/statutes) | [Français](/fr/statuts))
-
-## Ethical Source Working Group
+Ahead of a General Assembly and election of additional Committee members later this year, governance of the organization is the responsibility of its three Founding Members: Coraline Ada Ehmke, Don Goodman-Wilson, and Tobie Langel.
```

Comment:
```suggestion
Ahead of a [general assembly](/statutes#general-assembly) and election of additional committee members later this year, the [committee](/statutes#committee) is currently composed of the organization's founding members: Coraline Ada Ehmke, Don Goodman-Wilson, and Tobie Langel.
```

I think we should favor using lowercase instead of uppercase to avoid sounding too formal. I mistakenly used a lot of uppercase in the bylaws thinking this was common practice in the US. I've learnt since then that it was only required, even in legal documents, when needing to disambiguate a sentence, and usually avoidable by writing more clearly. I'd like us to adopt that style which I find more inclusive.

I also added links to relevant articles in the bylaws and reframed the sentence a bit to clarify that the committee already existed.